go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/ozonru/sarama-keycloak
```

Comment:
`go mod tidy`

Reply:
What do you mean?

Reply:
On my local machine after `go mod tidy` execution some of the indirect dependencies gone away, like:

```	
github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
github.com/k0kubun/pp v3.0.1+incompatible
github.com/mattn/go-colorable v0.1.6 // indirect
```

I like when there are no any exceeded dependencies :)

Reply:
removed